File: NeetCode/01-Arrays-Hashing/ValidAnagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        if (s.length() != t.length()) {
            return false;
        }
        char[] sArray = s.toCharArray();
        char[] tArray = t.toCharArray();

        Arrays.sort(sArray);
        Arrays.sort(tArray);

        return Arrays.equals(sArray, tArray);
    }
}
class Answer{
    public static void main(String[] args){
        Scanner scan = new Scanner(System.in);
        
        String s = scan.nextLine();
        String t = scan.nextLine();

        Solution solution = new Solution();
        solution.isAnagram(s,t);
        
        scan.close();
    }
}

File: NeetCode/01-Arrays-Hashing/ValidAnagram.cpp
#include <iostream>
#include <string>
#include <algorithm> 

using namespace std;

class Solution {
public:
    bool isAnagram(string s, string t) {
        if (s.length() != t.length()){
            return false;
        } 

        sort(s.begin(), s.end());
        sort(t.begin(), t.end());

        return s == t;
    }
};

int main() {
    Solution sol;
    string s;
    cin >> s;
    string t;
    cin >> t;

    if(sol.isAnagram(s, t)){
        cout << "true";
    }
    else{
        cout << "false";
    }

    return 0;
}